src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void draw();
+}
+
+
+
```

Comment:
your code contains a lot of redundant empty lines
please look through all the classes